packages/service-library/src/servicelib/aiohttp/application_setup.py

```diff
@@ -114,11 +140,13 @@ def app_module_setup(
     module_name: str,
     category: ModuleCategory,
     *,
+    # TODO: use settings_name as module_name!!
```

Comment:
is that still current?

Reply:
Right, this will come in a separate PR where I will completely remove all the config. There are some extra functionality coming and I preferred doing it separately (this way I do not need to redo the tests)